updating to version 2.3.11

ous8292 opened the pull request:
Updating unixODBC from version 2.3.9 to version 2.3.11

skupriienko commented:
On osx-64:
`__handles.c:1809:20: error: use of undeclared identifier 'CLOCK_REALTIME'`

skupr-anaconda commented:
Try to add a local `cbc.yaml` to the recipe like this one https://github.com/AnacondaRecipes/pymssql-feedstock/blob/a7dfa13dfdd55548aeac10923407aae6bda58b93/recipe/conda_build_config.yaml#L1-L6. 
It seems like currently we use -DCMAKE_OSX_SYSROOT=/opt/MacOSX10.10.sdk and it doesn't support `'CLOCK_REALTIME'`, see https://github.com/eclipse/paho.mqtt.c/issues/546
If it won't work then we will need to specify flags in `build.sh`

ous8292 commented:
`

> Try to add a local `cbc.yaml` to the recipe - this one https://github.com/conda-forge/unixodbc-feedstock/blob/main/recipe/conda_build_config.yaml It seems like currently we use -DCMAKE_OSX_SYSROOT=/opt/MacOSX10.10.sdk and it doesn't support `'CLOCK_REALTIME'`, see [eclipse/paho.mqtt.c#546](https://github.com/eclipse/paho.mqtt.c/issues/546)
> 
> And update `build.sh` from here https://github.com/conda-forge/unixodbc-feedstock/blob/main/recipe/build.sh:
> 
> ```
> #!/bin/bash
> 
> set -e
> set -x
> 
> export CFLAGS="-O2 ${CFLAGS}"
> export CXXFLAGS="-O2 ${CXXFLAGS}" 
> 
> autoreconf -vfi
> 
> ./configure --sysconfdir=/etc --prefix=$PREFIX --build=${BUILD}
> 
> make -j${CPU_COUNT} ${VERBOSE_AT}
> make install
> ```

I added the above as mentioned, but still getting the error.

@varlackc and I were doing some digging and came the line we think might be breaking it.

in the file The file name is `__handles.c` - line `1809` - `clock_gettime( CLOCK_REALTIME, &waituntil );`. 

in the file location - `../miniconda3/envs/c3i-test/conda-bld/unixodbc_1659124727489/work/DriverManager`

skupr-anaconda commented on recipe/meta.yaml:
```diff
@@ -1,6 +1,6 @@
+{% set sha256 = "d9e55c8e7118347e3c66c87338856dad1516b490fb7c756c1562a2c267c73b5c" %}
 
 package:
   name: {{ name|lower }}
```
Could you please fix the license name as `LGPL-2.1-or-later` (the license file is inside http://www.unixodbc.org/unixODBC-2.3.11.tar.gz).
The comment:
```
    # patches:
    #   - 0001-Add-support-for-editline.patch
```
can be removed.

ous8292 commented:
Updating unixODBC from version 2.3.9 to version 2.3.11

Changes:
- bumped version number
- updated SHA
- added conda-build-config file
- updated license
- added lines to build.sh

varlackc reviewed (commented):
we need a `dev_url`, the following `url` seems to be mentioned in the documentation in the download section:
https://github.com/lurcher/unixODBC

varlackc reviewed (approved):
LGTM
